Сетка страниц и крупных блоков на флексах

Savokivon opened the pull request:
---
:mortar_board: [Сетка страниц и крупных блоков на флексах](https://up.htmlacademy.ru/htmlcss/34/user/2128385/tasks/7)

:boom: https://htmlacademy-htmlcss.github.io/2128385-sedona-34/5/

Savokivon committed:
Стилизация главной страницы

Savokivon committed:
Сетка каталога

Savokivon committed:
Цвет ссылок

jumile commented on catalog.html:
```diff
@@ -8,217 +8,240 @@
+        <h1 class="sedona-hotels">Гостиницы Седоны</h1>
+        <header class="inner-header">
+          <h1 class="visually-hidden">Каталог</h1>
+          <ul class="breadcrumbs">
```
Для хлебных крошек лучше использовать нумерованный список - ведь порядок страниц здесь важен, не так ли? 
И еще этому списку надо убрать отступы.

jumile commented on index.html:
```diff
@@ -8,131 +8,145 @@
+      <section class="sedona">
+        <img class="welcome" src="images/welcome-sedona.png" alt="Добро пожаловать" width="458" height="352">
+        <h1 class="visually-hidden">Седона</h1>
+        <h2 class="title">Седона — небольшой городок в Аризоне, заслуживающий большего!</h2>
```
Обратите внимание, что заголовок "Седона - небольшой городок…" и текст про 5 причин должны быть на белом фоне, а у Вас они получаются на фоне фотографии. Здесь лучше этот текст выделить в отдельный блок, который не будет связан с тем, у которого стоит фон-фотография.

jumile commented on styles/styles.css:
```diff
@@ -14,7 +14,15 @@
+  height: 100%;
+}
+
 body {
```
Поправьте цвет фона для всей страницы - посмотрите в макете, он  не белый.

jumile commented on index.html:
```diff
@@ -8,131 +8,145 @@
+      <ul class="contacts-container">
+        <li>
+          <ul class="contacts">
+          <a class="vk" href="https://vk.com/htmlacademy" target="_blank">
```
Внутри ul обязательно должен быть li, пункт списка. Остальные элементы вкладываются уже в li, поэтому тег a здесь неправомерен, код не пройдет проверку на валидаторе. 
И здесь еще путаница с закрытием тегов li/ul. Посмотрите теорию вот здесь и проверьте у себя: https://up.htmlacademy.ru/htmlcss/34/module/2/item/1/3

jumile commented on index.html:
```diff
@@ -8,131 +8,145 @@
+            <p class="reason">Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
+          </div>
+        </li>
+        <ul class="advantages">
```
Здесь неверный порядок тегов. Внутри ul дочерними могут быть только li. Остальные элементы вкладываются уже в li. Т.е. напрямую вложить один список в другой нельзя, список можно вложить внутрь пункта другого списка.
Посмотрите структуру вложенных списков в учебнике - раздел Вложенные списки, обратите внимание именно на html-код, на то, как теги вложены друг в друга. https://up.htmlacademy.ru/htmlcss/34/book/6.11

jumile reviewed (changes requested):
Света, вот прям хорошо! ЗдОрово, что справились с липким футером с первого раза.
Не торопитесь с внутренними элементами пока.
Осталось только поправить разметку списков - почему-то возникли в ней ошибки, и задание будет выполнено. Вы можете проверять код на валидаторе - validator.w3.org, чтобы находить ошибки в разметке.

Savokivon committed:
Исправляет ошибки

Исправлены вложенные списки
Заголовок вынесен в отдельный блок
Хлебные крошки - нумерованный список

Savokivon committed:
Исправляет цвет фона

Savokivon committed:
Вносит мелкие исправления

Цвет фона
Подключение шрифтов
Блок с контактами
